language/German/strings.po

```diff
@@ -3316,43 +3374,43 @@ msgstr "Drucker\tFehlerhaft\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2969
 msgid "Mag. Tape\tUnimplemented\n"
-msgstr ""
+msgstr "Magnetband\tNicht implementiert\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2971
 msgid "Mag. Tape\tImperfect\n"
-msgstr ""
+msgstr "Magnetband\tFehlerhaft\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2974
 msgid "Punch Tape\tUnimplemented\n"
-msgstr ""
+msgstr "Lochstreifen\tNicht implementiert\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2976
 msgid "Punch Tape\tImperfect\n"
-msgstr ""
+msgstr "Lochstreifen\tFehlerhaft\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2979
 msgid "Mag. Drum\tUnimplemented\n"
-msgstr ""
+msgstr "Magnettrommel\tNicht implementiert\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2981
 msgid "Mag. Drum\tImperfect\n"
-msgstr ""
+msgstr "Magnettrommel\tFehlerhaft\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2984
 msgid "(EP)ROM\tUnimplemented\n"
-msgstr ""
+msgstr "(EP)ROM\tNicht implementiert\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2986
 msgid "(EP)ROM\tImperfect\n"
-msgstr ""
+msgstr "(EP)ROM\tFehlerhaft\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2989
 msgid "Communications\tUnimplemented\n"
-msgstr ""
+msgstr "Kommunikation\tNicht implementiert\n"
```

Comment:
I guess what it means is "Kommunikationsgerät"? But "Kommunikation" is probably fine too